c2corg_ui/static/js/editing/imageediting.js

```diff
@@ -120,4 +120,54 @@ app.ImageEditingController.prototype.filterImageTypes = function(imageTypes) {
   return imageTypes.filter(removeCopyright);
 };
 
+/**
+ * @param {appx.Document} doc Document attributes.
+ * @override
+ * @public
+ */
+app.ImageEditingController.prototype.PreSetQuality = function(doc) {
+  var score = 0;
+
+  if ('title' in doc.locales[0] && doc.locales[0]['title']) {
+    score = score + 1;
+  }
+  
+  if ('description' in doc.locales[0] && doc.locales[0]['description']) {
+    score = score + 1;
+  }
+ 
+  if ('geometry' in doc && doc['geometry']) {
+    const geometry = doc['geometry'];
+    if ('geom' in geometry && geometry['geom']) {
+      score = score + 1;
+    }
+  }
+ 
+  if ('activities' in doc && doc['activities'].length > 0) {
+    score = score + 0.5;
+  }
+  
+  if ('categories' in doc && doc['categories'].length > 0) {
+    score = score + 0.5;
+  }
+ 
+  if ((doc['associations']['waypoints'].length > 0 ) ||
```

Comment:
useless parentheses (I don't think removing them would make the statement less obvious)